test/send-anon-metrics.js

```diff
@@ -0,0 +1,68 @@
+var test = require('tap').test
+
+var common = require('./lib/common.js')
+var client = common.freshClient()
+var server = require('./lib/server.js')
+
+function nop () {}
+
+var URI = 'https://npm.registry:8043/-/npm/anon-metrics/v1/:metricId'
```

Comment:
`:metricId` shouldn't be passed in literally, it's supposed to be replaced with the `metricId` value from the document we're given.